lib/entities/context/index.ts

```diff
@@ -109,8 +113,14 @@ export function mergeRequests(base: QuoteRequest, layer: QuoteRequest): QuoteReq
     const config = Object.assign({}, baseConfig, {
       // if base does not specify simulation params but layer does, then we add them
       simulateFromAddress: baseConfig.simulateFromAddress ?? layerConfig.simulateFromAddress,
-      deadline: baseConfig.deadline ?? layerConfig.deadline,
-      recipient: baseConfig.recipient ?? layerConfig.recipient,
+      // NOTE: This is agreed upon between cross team,
+      // FE doesn't pass the deadline for classic config only quote request,
+      // so in that case URA passes down the hardcoded deadline
+      // FE only passes recipient down, if the wallet is connected,
+      // so in case the wallet is connected, and if it's a pure classic config only quote request,
+      // FE still doesn't pass the recipient, so URA passes down the hardcoded recipient
```

Comment:
This is a large change to the current behavior **and** we have to be extra careful that a user never submits a quote where the recipient is uniswap.eth because they they would lose their tokens. What's the motivation for this?

Reply:
Also nit but can we condense this comment into just saying that we use the default values if none are provided?